test/TransformerEsbuild.js

```diff
@@ -0,0 +1,61 @@
+// Compiles JavaScript with esbuild.
+//
+// We use this because esbuild-jest hasn't been updated in two years.
```

Comment:
You might be able to use ts-jest; it looks like ts-jest recently added support for esbuild as an experimental feature. Though I'm not clear if this support is working yet. See:
- https://kulshekhar.github.io/ts-jest/docs/guides/esm-support#use-esm-presets
- https://github.com/kulshekhar/ts-jest/search?q=esbuild
- https://github.com/kulshekhar/ts-jest/issues/2135

Reply:
ts-jest does not support esbuild based transpilation
you could use swc-jest, we use at the company to run around 5k tests and it is pretty fast and awesome